tests/components/zone/test_init.py

```diff
@@ -1,229 +1,224 @@
-        assert len(self.hass.states.entity_ids("zone")) == 0
-
-    def test_setup_registered_zone_skips_configured_zone(self):
-        """Test if config entry will override configured zone."""
-        entry = MockConfigEntry(domain=zone.DOMAIN, data={zone.CONF_NAME: "Test Zone"})
-        entry.add_to_hass(self.hass)
-        info = {"name": "Test Zone", "latitude": 1.1, "longitude": -2.2}
-        assert setup.setup_component(self.hass, zone.DOMAIN, {"zone": info})
-
-        assert len(self.hass.states.entity_ids("zone")) == 1
-        state = self.hass.states.get("zone.test_zone")
-        assert not state
-
-    def test_active_zone_skips_passive_zones(self):
-        """Test active and passive zones."""
-        assert setup.setup_component(
-            self.hass,
-            zone.DOMAIN,
-            {
-                "zone": [
-                    {
-                        "name": "Passive Zone",
-                        "latitude": 32.880600,
-                        "longitude": -117.237561,
-                        "radius": 250,
-                        "passive": True,
-                    }
-                ]
-            },
-        )
-        self.hass.block_till_done()
-        active = zone.async_active_zone(self.hass, 32.880600, -117.237561)
-        assert active is None
-
-    def test_active_zone_skips_passive_zones_2(self):
-        """Test active and passive zones."""
-        assert setup.setup_component(
-            self.hass,
-            zone.DOMAIN,
-            {
-                "zone": [
-                    {
-                        "name": "Active Zone",
-                        "latitude": 32.880800,
-                        "longitude": -117.237561,
-                        "radius": 500,
-                    }
-                ]
-            },
-        )
-        self.hass.block_till_done()
-        active = zone.async_active_zone(self.hass, 32.880700, -117.237561)
-        assert "zone.active_zone" == active.entity_id
-
-    def test_active_zone_prefers_smaller_zone_if_same_distance(self):
-        """Test zone size preferences."""
-        latitude = 32.880600
-        longitude = -117.237561
-        assert setup.setup_component(
-            self.hass,
-            zone.DOMAIN,
-            {
-                "zone": [
-                    {
-                        "name": "Small Zone",
-                        "latitude": latitude,
-                        "longitude": longitude,
-                        "radius": 250,
-                    },
-                    {
-                        "name": "Big Zone",
-                        "latitude": latitude,
-                        "longitude": longitude,
-                        "radius": 500,
-                    },
-                ]
-            },
-        )
+    assert await setup.async_setup_component(hass, zone.DOMAIN, {"zone": info})
+
+    assert len(hass.states.async_entity_ids("zone")) == 2
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4870504)